apps/tup-cms/src/apps/user_news/templates/user_news/full_list.html

```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+
+<div class="container">
+    <section class="c-news c-news--list">
+      {% include './includes/content_styles.html' %}
+  
+      {% with title_tag="h3" subtitle_tag="h4" %}
+      {% include './includes/list.html' %}
+      {% endwith %}
+  
+    </section>
+</div>
```

Comment:
**5. Unsuitable markup.** Remove the container (and re-indent afterward). The container here is adding unexpected margin around the list. Such margin is already provided around the page content. [^1]

<details><summary>I.e. this suggestion diff.</summary>

```suggestion
<section class="c-news c-news--list">
  {% include './includes/content_styles.html' %}
  
  {% with title_tag="h3" subtitle_tag="h4" %}
  {% include './includes/list.html' %}
  {% endwith %}
</section>
```

</details>

| Existing Margin (from page Container) | Undesired Margin (from plugin Container) |
| - | - |
| <img width="1194" alt="existing" src="https://github.com/TACC/tup-ui/assets/62723358/008f4cb5-fd71-49b0-be6c-606150701ab1"> |  <img width="1194" alt="erroneous" src="https://github.com/TACC/tup-ui/assets/62723358/e4d9321d-80b0-4215-84d9-a2103fc03dd3"> |

[^1]: A container is used (on our sites) to wrap **all** content of a page, **not** a single block of content within a page that has multiple blocks. But this container is doing the latter.

Reply:
Fixed.